client/reader/start/style.scss

```diff
@@ -0,0 +1,281 @@
+.reader-start {
+	position: relative;
+
+	.reader-start__intro {
+
+		h1 {
+			color: darken( $gray, 30% );
+			font-size: 30px;
+			font-weight: 300;
+			margin-bottom: 8px;
+
+			@include breakpoint( "<960px" ) {
+				padding: 0 15px;
+			}
+
+			@include breakpoint( "<660px" ) {
+				margin-top: 20px;
+			}
+		}
+
+		p {
+			color: $gray;
+			font-size: 15px;
+			font-weight: 100;
+			line-height: 1.4;
+			margin-bottom: 15px;
+
+			@include breakpoint( "<960px" ) {
+				padding: 0 15px;
+			}
+
+			@include breakpoint( ">480px" ) {
+				font-size: 18px;
+			}
+		}
+	}
+
+	.reader-start__cards {
+		display: flex;
+ 		flex-wrap: wrap;
+  		margin-top: -14px;
+
+  		// Aligns last card with grid
+  		&:after {
+  			content:'';
+  			flex: auto;
+  			width: 46%;
+  		}
+	}
+}
+
+.site-icon {
```

Comment:
This is breaking site-icons, should only be styled within one of your custom components.